test/stackanalysismodule.test.ts

```diff
@@ -47,4 +52,54 @@ suite('stacknalysis module', () => {
         expect(context.globalState.get('f8_3scale_user_key')).equals('');
     });
 
+    test('stack_collector should call the callback', () => {
+        let callback = sandbox.spy();
+        stackanalysismodule.stack_collector('', '123', callback);
+        expect(callback).calledOnce;
+    });
+
+    test('get_stack_metadata should call the callback when called with invalid fileuri', () => {
+        let callback = sandbox.spy();
+        sandbox.spy(vscode.window, 'showErrorMessage');
```

Comment:
we should also test that this spy is called calledOnce
Like `const showErrorMessageSpy = sandbox.spy(vscode.window, 'showErrorMessage');
stackanalysismodule.get_stack_metadata(context, 'some/path/', callback);
expect(showErrorMessageSpy ).calledOnce;
`

Reply:
This we are doing right , see line number 66

 `expect(vscode.window.showErrorMessage).calledOnce;`